dynawo/3rdParty/libiidm/core/include/IIDM/components/Bus.h

```diff
@@ -50,17 +50,17 @@ class Bus: public Identifiable, public ContainedIn<VoltageLevel> {
   VoltageLevel & voltageLevel() { return parent(); }
 
 public:
-  /** tells if the tension is set */
+  /** tells if the voltage is set */
```

Comment:
This library is about to be deleted, not sure it is useful to update it... Same for others 3rd parties.
What do you think @gautierbureau ?

Reply:
Personally I think that 3rd parties shouldn't be corrected - they are 3rd parties that we include as they are.

Reply:
These are minor spelling and translation corrections, which do not substantially alter the third parties.

Reply:
I have restored the 3rdParty directory.